.travis.yml

```diff
@@ -5,7 +5,7 @@ python:
 
 install:
   - pip install --upgrade pip
-  - pip install -U -r python/requirements.txt
+  - pip install -U -r python/requirements.txt dask[dataframe] distributed
```

Comment:
I suspect you don't want Dask as a required dependency, so I didn't add it to requirements.txt.